softgroup/basis/common-router/src/main/java/com/softgroup/common/router/api/AbstractRequestHandler.java

```diff
@@ -6,7 +6,7 @@
 import com.softgroup.common.protocol.Response;
 import com.softgroup.common.protocol.ResponseData;
 
-public class AbstractRequestHandler<T extends RequestData, R extends ResponseData> implements RequestHandler {
+public abstract class AbstractRequestHandler<T extends RequestData, R extends ResponseData> implements RequestHandler {
 	@Override
 	public String getName() {
```

Comment:
After making its abstract i can clear this meaningless code (getName, handle), but untill heirs dont realize real handle method i propose return null by default in AbstractRequestHandler.
I aggre to delete getName method.